spec/requests/odata/root_json_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "root json" do
+  include_context "odata"
+
+  let(:path) { mission_api_route }
+
+  context "with no forms" do
+    it "renders as expected" do
+      expect_output({
+        "@odata.context": "http://www.example.com/en/m/#{get_mission.compact_name}/odata/v1/$metadata?mode=m",
+        value: []
+      }.to_json)
+    end
+  end
+
+  context "with several forms" do
+    include_context "odata_with_forms"
+
+    it "renders as expected" do
```

Comment:
Preferrably we'd have two forms that actually get included here, just to show the 2 case.